### PR TITLE
Fix icons & labels in vertical tabs

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Various fixes for the admin form:

- Fixes icons in vertical tabs
- Fixes the live-update of contact label in vertical tabs
- Fixes extraneous question-marks appearing in the "Additional Settings" tab description
- Fixes a few jshint warnings

Before
----------------------------------------
Icons missing
Typing into the contact label field doesn't update the tab label until you hit "save"
Extraneous question-marks appear in the "Additional Settings" tab description

After
----------------------------------------
Fixed

### CODE REVIEW

**Comment by KarinG** [comment]
@colemanw -> we have replaced the icons with fontawesome icons -> 

image

File: js/webform_civicrm_admin.js [hunk 1]
```diff
@@ -451,11 +451,11 @@ var wfCiviAdmin = (function ($, D) {
             if (cl[1] == 'icon') {
               var icon_name = cl[2];
               if (cl[0] == 'contact') {
-                name = 'name="' + (i + 1) + '_contact_type"'
+                name = 'name="' + (i + 1) + '_contact_type"';
                 var type = $('select[name="' + (i + 1) + '_contact_type"', '#webform-civicrm-settings-form').val();
                 icon_name = getContactIcon(type);
               }
-              $('#webform-civicrm-settings-form .vertical-tabs__menu-item a').eq(i).children('span').prepend('<i class="crm-i ' + icon_name + '" ' + name + '> </i>');
+              $('#webform-civicrm-settings-form .vertical-tabs__menu-item').eq(i).children('a').prepend('<i class="crm-i ' + icon_name + '" ' + name + '> </i>');
```
Comment (colemanw) [inline js/webform_civicrm_admin.js:453]: @KarinG yes I see on this line you are using `crm-i` fontawesome icons. But when I tested on a fresh copy of D9 these icons did not actually show up without this patch.

Comment (KarinG) [reply js/webform_civicrm_admin.js:453]: So very odd... are you on D9.3? I'm looking at a D9.2.9 site and can see the icons. 

I'm going to test this manually as we don't have much test coverage yet for the admin form.

Comment (colemanw) [reply js/webform_civicrm_admin.js:453]: Looks like `civicrm-buildkit` gave me Version 9.1.15.
I'm not sure why that would make a difference though.
At any rate this change ought to make the jquery less brittle so if the icons still work for you after as well as before, it's probably good to merge.

Comment (KarinG) [reply js/webform_civicrm_admin.js:453]: I'm still going to test it - I'm testing everything!

**Comment by KarinG** [comment]
Screenshot from test before this PR -> 

image

**Comment by KarinG** [comment]
Confirmed this is working -> I can change Contact Type -> Organization e.g. and it updates the label/tab on the left without having to save it first ✅

image

**Comment by KarinG** [comment]
@colemanw -> one ad-hoc issue you may be able to help us out with [this was already like this prior to this PR] -> when you hover over a tab on the left -> e.g. activities -> the fontawesome icon disappears -> 

image

**Comment by KarinG** [comment]
Screenshot from test after merging this PR -> 

image

**Comment by colemanw** [comment]
> one ad-hoc issue you may be able to help us out with [this was already like this prior to this PR] -> when you hover over a tab on the left -> e.g. activities -> the fontawesome icon disappears ->

I'm not seeing that on my local site - is it still an issue after this PR?

**Comment by KarinG** [comment]
Yes - I noticed that when I ran my live site check (post merge) on webform-civicrm.io - let me make coffee and then clear caches some more.

**Comment by KarinG** [comment]
I blew away webform_civicrm and recloned -> on hover -> icon still goes missing. This is on webform-civicrm.io - I'll dm you a drush uli!

image

**Comment by KarinG** [comment]
Ah - this only happens with Claro (admin-theme). Using Seven -> all is well -> 
image

**Comment by colemanw** [comment]
I don't find that quirk particularly concerning if it's just limited to one theme.

**Comment by KarinG** [comment]
Yes - except it’s the new Drupal Core Admin theme. Is there anything obvious you can spot with your JS eyes?

**Pull request merged**